RELEASE.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Release process automation script. 
+# Used to create branch/tag, update VERSION files and and trigger release by force pushing changes to the release branch 
+
+# set to 1 to actually trigger changes in the release branch
+TRIGGER_RELEASE=0 
+
+#REPO=git@github.com:eclipse/che-machine-exec
+#VERSION=7.7.0
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-t'|'--trigger-release') TRIGGER_RELEASE=1; shift 0;;
+    '-r'|'--repo') REPO="$2"; shift 1;;
+    '-v'|'--version') VERSION="$2"; shift 1;;
+  esac
+  shift 1
+done
+
+usage ()
+{
+  echo "Usage: $0 --repo [GIT REPO TO EDIT] --version [VERSION TO RELEASE] [--trigger-release]"
+  echo "Example: $0 --repo git@github.com:eclipse/che-machine-exec --version 7.7.0 --trigger-release"; echo
+}
+
+if [[ ! ${VERSION} ]] || [[ ! ${REPO} ]]; then
+  usage
+  exit 1
+fi
+
+# derive branch from version
+BRANCH=${VERSION%.*}.x
+
+# work in tmp dir
+TMP=$(mktemp -d); mkdir -p "$TMP"; cd "$TMP" || exit 1
```

Comment:
`mkdir -p "$TMP"` seems to be unnecessary here, since `mktemp` will create it.

Maybe also a `pushd` with a trap to `popd` on exit makes sense here?

Reply:
fair point about mktemp... but no need to popd back since if the dir isn't created, we can't have pushd'd or cd'd into it.

Reply:
Doesn't the script leave your shell in `/tmp` upon completion though?

Reply:
no, it runs in its own subshell